src/Components/CreateImageWizard/steps/ImageOutput/TargetEnvironment.tsx

```diff
@@ -191,13 +192,18 @@ const TargetEnvironment = () => {
           >
             <Checkbox
               label="VMware vSphere"
-              isChecked={
-                environments.includes('vsphere') ||
-                environments.includes('vsphere-ova')
-              }
+              isChecked={hasVsphere}
               onChange={() => {
-                setHasVSphere(!hasVSphere);
-                handleToggleEnvironment('vsphere-ova');
+                if (!hasVsphere) {
+                  handleToggleEnvironment('vsphere-ova');
+                } else {
+                  if (environments.includes('vsphere')) {
+                    handleToggleEnvironment('vsphere');
+                  }
+                  if (environments.includes('vsphere-ova')) {
+                    handleToggleEnvironment('vsphere-ova');
+                  }
+                }
```

Comment:
This block is a bit confusing to me, and I had to think twice. I think in this context, maybe this will be a bit more readable:
!environments.includes('vsphere') && !environments.includes('vsphere-ova')

Could you explain a bit more what is this supposed to do - why do we call handleToggleEnvironment('vsphere-ova') if !hasSphere?

Reply:
The `if (!hasVsphere)` handles the case where a vSphere environment has not been selected, and then selects vsphere-ova as the default.

The `else` clause handles the case where a vSphere environment has been selected, and deselects and also deselects whichever radio (ova or vmdk) has been selectged.

I think you're right, @avitova, that we should make this more obvious. I think the right way to do this is to refactor handleToggleEnvironment into two separate functions, `addEnvironment` and `removeEnvironment`. I'll create an issue to track this refactoring.

Reply:
Fixed it up a bit @avitova, lmk if you still find it confusing and we can do some additional refactoring.

Reply:
No, it looks good, thank you!:)